README.md

```diff
@@ -2,32 +2,50 @@
 
 ## Building
 
-Different compilers can be set by specifying `CMP`, e.g. `make CMP=intel`
-to build with Intel compilers, see `Makefile` for options.
-
-By default an optimised library will be built, debugging versions of the
-library can be built with `make BUILD=debug`, a development version which 
-additionally sets compile time flags to catch coding errors can be built 
-with `make BUILD=dev` (GNU compilers only currently). The behavior of debug
-and development versions of the library can be changed before the initialization
-using the variable ``decomp_debug`` or the environment variable ``DECOMP_2D_DEBUG``.
+The build system is driven by `cmake`, to generate a build system run
```

Comment:
I will push a commit for the README, this should be updated.

Reply:
See https://github.com/xcompact3d/2decomp-fft/pull/149

Reply:
Makefile will be removed after merging

Reply:
When CMake is ready and merged, it is probably best to keep both the Makefile and CMake for a small period of time to allow for a smooth transition.

Reply:
agreed - I've lifted into a separate issue. I think only need to check ifx and this should be good to go